evalml/automl/automl_search.py

```diff
@@ -811,7 +823,7 @@ def describe_pipeline(self, pipeline_id, return_dict=False):
             logger.info(all_objective_scores)
 
         if return_dict:
-            return pipeline_results
+            return automl_dict
```

Comment:
Why do we create `automl_dict` rather than just return `pipeline_results` like we did before? Is `pipeline_results` not a dictionary already?

Reply:
No reason! In previous impl I needed to add something new but I ended up adding to `pipeline_results` instead so the two are the same again. I'll remove :)